api/swagger-spec/oapi-v1.json

```diff
@@ -24841,6 +24841,10 @@
       "$ref": "v1.ObjectMeta",
       "description": "Standard object's metadata."
      },
+     "postProcessMessage": {
+      "type": "string",
+      "description": "Optional: An instructional message for the user processing this template (escaped parameters will be substitued)."
```

Comment:
not to worry, we have a large contingent of high-functioning OCD sufferers in this org.